packages/web/src/utils/devserver.config.ts

```diff
@@ -33,6 +34,18 @@ export function getDevServerConfig(
     serveOptions,
     buildOptions
   );
+  webpackConfig.plugins = [
```

Comment:
I don't think this is required, since we already set the values here:
https://github.com/nrwl/nx/blob/master/packages/web/src/utils/config.ts#L75

Reply:
Thanks, You're  right. I didn't see that.